index.dev.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
Leaving this file here, since we'll want to get back to developing with this file that loads each thing independently.